src/debug/ee/controller.cpp

```diff
@@ -4957,6 +4958,7 @@ DebuggerBreakpoint::DebuggerBreakpoint(Module *module,
 
     BOOL bindAcrossAllJittedInstances = !native;
     MethodDesc* pGenericInstanceFilter = NULL;
+
 #ifdef DEBUG
```

Comment:
You can get rid of this debug code now. I used it because there was no public API to achieve the same result but now that you are adding one we don't need it.